examples/matrix_multiplier/tests/test_matrix_multiplier.py

```diff
@@ -89,7 +90,9 @@ async def test_multiply(dut, a_data, b_data):
     in_monitor = MatrixInMonitor(dut, callback=expected_output.append)
     out_monitor = MatrixOutMonitor(dut)
 
-    scoreboard = Scoreboard(dut)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        scoreboard = Scoreboard(dut)
```

Comment:
How much work would it be to replace this with "something else". I'm not sure what that would be. Too much for this PR?

Reply:
I'll look into it.